### ISSUE TITLE
Provide a method in the simulator API that returns the metadata level

### ISSUE THREAD

**Issue by bzinberg** [NextCenturyCorporation/MCS#234]
In our TA1 codebase we have the [function](https://github.com/probcomp/GenAgent.jl/blob/2384c21ad7a4c99d0c39cb9775ff6066486bea26/src/state/agent.jl#L9-L18)

```julia
function infer_information_level(step_metadata::PyObject)::String
    if length(step_metadata.object_list) > 0 || length(step_metadata.structural_object_list) > 0
        return "oracle"
    elseif length(step_metadata.object_mask_list) > 0
        return "level2"
    else
        return "level1"
    end
end
```

This logic is a translation into code of information that was communicated to us a while back, not sure when or where.  We have no way to ensure this function stays up to date as the API for the simulator changes.  AFAIK, it would be much better to move "what is the metadata level" code to the simulator, where it would be just a field accessor.

**Comment by rartiss55** [comment]
get_metadata_level() has been added to the controller class.

Merged to development and will be included in next release (0.4.2).

**Comment by bzinberg** [comment]
Thanks @rartiss55 for the work and for the update!
